extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/kernel/StageDefinition.java

```diff
@@ -122,6 +124,7 @@
     this.maxWorkerCount = maxWorkerCount;
     this.shuffleCheckHasMultipleValues = shuffleCheckHasMultipleValues;
     this.frameReader = Suppliers.memoize(() -> FrameReader.create(signature))::get;
+    this.maxInputBytesPerWorker = maxInputBytesPerWorker;
```

Comment:
The stage definition is sent over the wire. In a rolling upgrade, what happens if an older controller sends a task without this value? Should we use a `Long` and use the default if the value is null (as it will be if the worker task is missing this field)?

Reply:
Thanks for pointing this out, changed to Long